test/TestProjects/Accessibility-LowLevel-NoAuth/Generated/AccessibilityClient.cs

```diff
@@ -25,17 +25,17 @@ public partial class AccessibilityClient
         /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
         public virtual HttpPipeline Pipeline => _pipeline;
 
-        /// <summary> Initializes a new instance of AccessibilityClient for mocking. </summary>
-        protected AccessibilityClient()
```

Comment:
Why is the protected constructor for mocking removed?

Reply:
This client constructor does not have required parameters. So we have to choose between 

![image](https://user-images.githubusercontent.com/97225342/160979783-a43356d7-b619-4121-9a12-d23d6436f603.png)

Reply:
This is ok for a test case, but it would be a [violation of the guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-client-constructor-for-mocking) if it ever appeared in a public API.

Reply:
So, we remove the public one?

Reply:
It is ok for a test case and this is a test case, so no change is required.